src/Widgets/TerminalWidget.vala

```diff
@@ -149,6 +149,8 @@ namespace Terminal {
                                                          null, null).get_char ();
                     if (clicked_symbol != 0U) {
                         delta_cells += (clicked_row - current_row) * get_column_count ();
+                    } else {
```

Comment:
Could be simplified to:

```vala                                                    
if (clicked_symbol == 0U) {
    return Gdk.EVENT_STOP;
}

delta_cells = clicked_col - current_col + (clicked_row - current_row) * get_column_count ();
```

Reply:
OK, done.

Reply:
I think there is no reason to increment. You could simplify further:
‘‘‘
long delta_cells = clicked_col - current_col + (clicked_row - current_row) * get_column_count ();
‘‘‘